server/src/main/java/io/crate/metadata/cluster/AlterTableClusterStateExecutor.java

```diff
@@ -460,7 +460,7 @@ private ClusterState updateSettings(final ClusterState currentState, final Setti
                 indicesService.verifyIndexMetadata(updatedMetadata, updatedMetadata);
             }
         } catch (IOException ex) {
-            throw ExceptionsHelper.convertToElastic(ex);
+            throw new UncheckedIOException(ex);
```

Comment:
This will result in a `NotSerializableExceptionWrapper` if streamed now or? Doesn't matter as both, `NotSerializableExceptionWrapper` and `ElasticsearchException` will be unwrapped?

Reply:
Right thx for catching. Given that no tests failed I assumed it was already handled, but `writeException` only handles `IOException`.

Pushed a fixup. Not sure if it is a good solution. I would like to be ably to rely on built-in JDK exceptions being streamable, but also fine for me to drop that commit altogether.